src/main/groovy/com/dicedmelon/gradle/jacoco/android/JacocoAndroidUnitTestReportExtension.groovy

```diff
@@ -14,14 +14,18 @@ class JacocoAndroidUnitTestReportExtension {
        '**/Manifest*.*'].asImmutable()
 
   public static final Collection<String> butterKnifeExcludes =
-      ['**/*$ViewInjector*.*',
-       '**/*$ViewBinder*.*'].asImmutable()
+      ['**/*_ViewInjector*.*',
+      '**/*_ViewBinding*.*',
+       '**/*_ViewBinder*.*'].asImmutable()
 
   public static final Collection<String> dagger2Excludes =
-      ['**/*_MembersInjector.class',
+      ['**/*_MembersInjector*.class',
        '**/Dagger*Component.class',
        '**/Dagger*Component$Builder.class',
-       '**/*Module_*Factory.class'].asImmutable()
+       '**/*_*Factory*.*',
+       '**/*Component*.*',
+       '**/*Subcomponent*.*',
+       '**/*Module*.*'].asImmutable()
```

Comment:
These seem like common names that not only Dagger is using.  I know that at my company, we have a fair number of *Module.java files that definitely need coverage.  `Component` also seems uncommon, but not exclusive to Dagger.

Do you think it might be better to restrict these excludes to the buildDir/generated directory?